public/index.html

```diff
@@ -1098,13 +1133,35 @@ <h3 class="text-4xl lg:text-5xl text-center text-white">
                 </a>
 
                 <div
-                    class="order-first sm:order-last flex items-center gap-10 pb-14 sm:pb-0"
+                    class="order-first sm:order-last flex max-[315px]:flex-col items-center gap-10 pb-14 sm:pb-0"
                 >
                     <a
                         class="text-lg lg:hover:text-brand-green transition ease-linear"
                         href="mailto:day@zkwarsaw.dev"
                         >Contact Us</a
                     >
+
+                    <!-- facebook -->
```

Comment:
pls try to use separate PRs for separate features in the future

Reply:
Sure, thanks